spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -28,11 +40,16 @@
       it 'doesnt match a fragrant String' do
         expect(matcher.matches?(clean_code)).to be_falsey
       end
+
+      it 'reports the smells when should_not fails' do
+        expect(matcher.matches?(smelly_code)).to be_truthy
+        expect(matcher.failure_message_when_negated).to match('UncommunicativeVariableName')
```

Comment:
investigating and I didn't write this.  It is an artifact of the funky rebase - this file had huge merge conflicts and I did my best to get it into the right shape, but because my local master was behind the upstream master it all went a bit wrong - have removed this test